tensorrt_llm/__init__.py

```diff
@@ -13,12 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
+# Disable UCC to WAR allgather issue before DLFW 25.12 upgrade.
+os.environ["OMPI_MCA_coll_ucc_enable"] = "0"
```

Comment:
```suggestion
if os.environ.get("OMPI_MCA_coll_ucc_enable", "0") == "0":
    os.environ["OMPI_MCA_coll_ucc_enable"] = "0"
```

Would this be better? The user can still override `OMPI_MCA_coll_ucc_enable` if they know their UCC version is good.